R/get_paths.R

```diff
@@ -112,7 +112,9 @@ get_levelpaths <- function(x, override_factor = NULL, status = FALSE, cores = NU
   while(done < nrow(x) & checker < 10000000) {
     tail_topo <- outlets$topo_sort
 
-    pathIDs <- if(!is.null(cl)) {
+    pathIDs <- if(nrow(outlets) == 1) {
+      list(par_get_path(as.list(outlets), x, matcher, status))
```

Comment:
Had never tested this against a single super small basin as is done in the advanced network attributes vignette.